pinot-plugins/pinot-input-format/pinot-csv/src/main/java/org/apache/pinot/plugin/inputformat/csv/CSVMessageDecoder.java

```diff
@@ -181,9 +180,4 @@ public GenericRow decode(byte[] payload, GenericRow destination) {
       throw new RuntimeException("Error decoding CSV record from payload", e);
     }
   }
-
```

Comment:
Same as below: implementation could pass offset and length to ByteArrayInputStream instead of copying bytes.